designs/administrator/4.7/enable-dynamic-plugin-ui/index.md

```diff
@@ -0,0 +1,58 @@
+---
+parent: Administrator
+version: 4.7
+---
+
+# Enabling and disabling dynamic plugin UIs
+
+OpenShift console currently has static plugins that allow teams to contribute features to the UI such as OpenShift Virtualization and OpenShift Container Storage. These plugins live inside console code which means they can only be updated at the same cadence as the console. Newly developed ‘dynamic plugins’ will allow operators to contribute their included UIs at runtime, meaning that operators can update their UIs on their own cadence and more frequently.
+
+Admins installing operators which include dynamic plugins will need to opt-in to having their UIs included in the console UI, which can happen as the operator is being configured for install. Trusted operators could perhaps have this option included by default, whereas other not-explicitly-trusted operators would need to have the user explicitly opt in, as there could be a security risk.
+
+
+Once the operator with the dynamic plugin UI is installed, the user can also turn the dynamic plugin UI on or off in the operator details. The operator list view would also convey any operators with disabled dynamic UIs.
+
+## Enabling dynamic plugin UI during operator install
+
+### Trusted operator install
+
+![Operator trusted install](img/1-1-install-rh.png)
+- A new radio group would appear for operators that include dynamic plugin UIs.
+- Trusted operators would have their dynamic UIs enabled by default.
+- A note on “trusted”: Disconnected environments need all operator sources manually added so they all (even Red Hat & certified operators) will appear as “untrusted”.
+
+![Operator install popover](img/1-2-install-rhPop.png)
+- An info popover would further describe implications of enabling an operator’s UI.
+
+### Untrusted operator install
+
+![Operator untrusted install](img/1-3-install-3rd.png)
+- Untrusted operators would have dynamic UIs disabled by default.
+
+![Operator untrusted install alert](img/1-4-install-3rdEnable.png)
+- Enabling the UI for the untrusted operator shows an inline alert conveying implications.
+
+## Enabling/disabling dynamic plugin UIs in operator details
+
+![Operator details setting](img/3-1-details.png)
+- Operators with dynamic plugin UIs would now include a setting in their details that conveys if their dynamic UI is enabled or disabled.
```

Comment:
As with the list view below, the user needs to be able to `patch` the `Console` operator in order for the links (and possibly the entire `Console UI extension` section that contains them) to appear in the sidebar, no?  Being able to `get` the `Console` operator would allow us to show any plugins that are available, but not enable/disable them.